### PR TITLE
Issue 1336 command doc fix

### LINKED ISSUE #1336: optional command to run when the template is rendered: do not execute multiple commands
Hi everyone,
I would like to execute more that one command after templating. As detailed here under, I put a ';' in between the two commands to be executed.
Unfortunately, only the first one is run. When checking journalctl for nginxws service, it is never reloaded.
Thx for support.

### Consul Template version
consul-template v0.20.0 (b709612c)

### Configuration

```hcl

template {
  source = "/opt/application/nginxws/current/conf/nginxws.conf.tmpl"
  destination = "/opt/application/nginxws/current/conf/nginxws.conf"
  command = "chown nginxws: /opt/application/nginxws/current/conf/nginxws.conf;service nginxws reload || true"
}

### Debug output
2020/01/24 15:46:26.825170 [DEBUG] (runner) diffing and updating dependencies
2020/01/24 15:46:26.825179 [DEBUG] (runner) health.service(webcomws|passing,warning) is still needed
2020/01/24 15:46:26.825187 [INFO] (runner) executing command "chown nginxws: /opt/application/nginxws/current/conf/nginxws.conf; systemctl reload nginxws || true" from "/opt/application/nginxws/current/conf/nginxws.conf.tmpl" => "/opt/application/nginxws/current/conf/nginxws.conf"
2020/01/24 15:46:26.825217 [INFO] (child) spawning: chown nginxws: /opt/application/nginxws/current/conf/nginxws.conf
2020/01/24 15:46:26.826524 [DEBUG] (runner) watching 1 dependencies
2020/01/24 15:46:26.826538 [DEBUG] (runner) all templates rendered
2020/01/24 15:46:26.827177 [DEBUG] (cli) receiving signal "child exited"
2020/01/24 15:46:51.751634 [DEBUG] (runner) receiving dependency health.service(webcomws|passing,warn

### Expected behavior

Both commnad to be executed in serial: 
1- chown nginxws: /opt/application/nginxws/current/conf/nginxws.conf
2- service nginxws reload

### Actual behavior

Only first command is executed:
1- chown nginxws: /opt/application/nginxws/current/conf/nginxws.conf

### PR DESCRIPTION
**Pull request by eikenb**
People sometimes need to run >1 command when a template is rendered and don't want/can't just wrap it in a script and deploy that. This tells them how to do it.

Fixes #1336

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2467,6 +2473,28 @@ providing smooth integration with other Consul tools (like `consul maint` or
...
+command. That is you cannot join multiple commands with `&&`, `;`, `|`, etc.
+This is a restriction of how they are executed. **However** you are able to do
+this by combining the multiple commands in an explicit shell command using `sh
+-c`. This is probably best explained by example.
+
+Say you have a couple scripts you need to run when a template is rendered,
+`/opt/foo` and `/opt/bar`, and you only want `/opt/bar` to run if `/opt/foo` is
+successful. You can do that with the command...
+
+`command = "sh -c '/opt/foo && /opt/bar'"
```
Comment (findkim) [inline README.md:2488]: Can you add an ending back tick `?

File: README.md [hunk 2]
```diff
@@ -2467,6 +2473,28 @@ providing smooth integration with other Consul tools (like `consul maint` or
...
+
+As this is a full shell command you can even use conditionals. So accomplishes the same thing.
+
+`command = "sh -c 'if /opt/foo; then /opt/bar ; fi`"
```
Comment (findkim) [inline README.md:2492]: I believe this should be

`command = "sh -c 'if /opt/foo; then /opt/bar ; fi'"`

File: README.md [hunk 3]
```diff
@@ -2467,6 +2473,28 @@ providing smooth integration with other Consul tools (like `consul maint` or
...
+
+Using this method you can run as many shell commands as you need with whatever
+logic you need. Though it is suggested that if it gets to long you might want
```
Comment (findkim) [inline README.md:2495]: > too long

File: README.md (hunk 1 above)
Comment (eikenb) [reply README.md]: Done. Thanks!

File: README.md (hunk 2 above)
Comment (eikenb) [reply README.md]: Syntax highlighters have broken me. Fixed this as well.

File: README.md (hunk 3 above)
Comment (eikenb) [reply README.md:2495]: English fail.

**Pull request merged**